mkdocs.yml

```diff
@@ -86,11 +87,15 @@ extra:
 
 extra_css:
   - stylesheets/extra.css
+  - https://cdn.jsdelivr.net/npm/daisyui@4.4.23/dist/full.min.css
```

Comment:
Zou je niet beter de files kunnen inchecken? vaak wil je in productie geen cdn gebruiken.

Reply:
Ja, dat moeten we de doen mocht dit naar productie gaan. Dit was/is slechts een PoC. Ik ben er nog niet uit hoe we styling moeten doen, en of het bijvoorbeeld überhaupt slim is daisyui te gebruiken in deze context.

Reply:
Ja styling is nog een goed punt. Uiteindelijk moet het algoritmekader misschien ook over naar een andere template meer in overheidsstijl. Voor deze PoC denk ik prima het zo te doen

Reply:
Voor nu heb ik het in "Wireframe" theme gezet, om niet de indruk te wekken dat het af is. Daar zouden we wat aandacht aan moeten besteden.